### ISSUE TITLE
[v1.1] DeNas NLP Workflow TODOs

### ISSUE THREAD

**Issue by tianyil1** [intel/e2eAIOK#168]
Backgroud:
Since currently saving logits function will be triggered every epoch when enabled, which leads to poor performance in training.

current Workaround:
using two steps to enable - step1, enable saving logits to call denas nlp, process will exit after first save logits. step2, disable training logits for KD process.

Next step:
- [x] [Urgent] Add readme for customer to understand current issue and indicates the proper way of running
- [x] [improvement] Integrate saving logits and transfer learning into one stage process
